### PR TITLE
Remove old 37 transcripts from VEP (and cache files)

### PR DESCRIPTION
**Pull request by aparton**
No test has been added as we don't have a 37 test database.

This should remove both refseq_import and refseq_human_import transcripts.

### CODE REVIEW

File: modules/Bio/EnsEMBL/VEP/AnnotationSource/Database/Transcript.pm [hunk 1]
```diff
@@ -249,8 +249,8 @@ sub get_features_by_regions_uncached {
         # in human and mouse otherfeatures DB, there may be duplicate genes
         # skip those from analysis refseq_human_import and refseq_mouse_import
         next if $self->{core_type} eq 'otherfeatures' && $self->assembly !~ /GRCh37/i && $tr->analysis && $tr->analysis->logic_name =~ /^refseq_[a-z]+_import$/;
-
-        if($self->{core_type} eq 'otherfeatures' && defined($tr->display_xref)){
+	next if $self->{core_type} eq 'otherfeatures' && $self->assembly =~ /GRCh37/i && $tr->analysis && $tr->analysis->logic_name =~ /^refseq.+import$/;
```
Comment (at7) [inline modules/Bio/EnsEMBL/VEP/AnnotationSource/Database/Transcript.pm]: As you have already said  a comment on why all those checks are needed will be helpful

**Comment by aparton** [comment]
Due to the inclusion of the new RefSeq transcript set (mapped from 38) into the 37 otherfeatures database, we decided to remove all of the older, lower quality transcripts from the cache files. To do this, we filtered out all transcripts with the analysis logic_name 'refseq_import' or 'refseq_human_import'. All new transcripts have the logic name 'refseq_import_grch38'

**Comment by at7** [comment]
Can you please add the comment into the Transcript.pm file? Thanks

**Comment by at7** [comment]
merged into master and release/100

**Pull request merged**